src/main/java/seedu/address/logic/commands/CreateCommand.java

```diff
@@ -40,12 +41,22 @@ public CreateCommand(Accounts account) {
         newAccount = account;
 
         if (!UserAccountStorage.checkDuplicateUser(account.getUsername())) {
+            UserAccountStorage.addNewAccount(account.getUsername(), account.getPassword());
+            createIsSuccessful = true;
```

Comment:
would isCreatedSuccessfully be better for the boolean variable?

Reply:
would it be better to move lines 43 to 49 to the execute command instead of the constructor?